### DISCUSSION TITLE
Wcfm Ultimate Seller Badges

### DISCUSSION THREAD

**Discussion by usamaahmed5656** [inspireui/support#5624] [New Feature/Ideas Requests]
the app should have to be compatible with wcfm ultimate addon ..which has a option to assign seller badges to store ...this will help help us to give badges to store which is performing well ...Badges like Verified seller etc

**Comment by Yan-Aleman** [comment]
> the app should have to be compatible with wcfm ultimate addon ..which has a option to assign seller badges to store ...this will help help us to give badges to store which is performing well ...Badges like Verified seller etc

Cool feature, much needed.

**Reply by usamaahmed5656** [reply]
please also check this thread (Live streaming for all wcfm stores listed https://github.com/inspireui/support/discussions/5362)

**Comment by usamaahmed5656** [comment]
@Yan-Aleman    

please also check this thread (Live streaming for all wcfm stores listed #5362)
